### ISSUE TITLE
fix: Rewrite "No metadata found" error more clearly, adding explanation that perhaps a using codebase is implemented but no flags have been written

### ISSUE THREAD

**Issue by capndave** [typestack/class-validator#1124]
### Description
`src/validation/ValidationExecutor.ts` conditionally returns the following error: `No metadata found. There is more than once class-validator version installed probably. You need to flatten your dependencies.` There are two issues with this error.
- the error has typos/incorrect grammar. "There is more than once class-validator version installed probably" should instead read "There are probably more than one class-validator versions installed", or "There are probably multiple class-validator versions installed"
- This error sometimes arises when a developer uses a module as a dependency that itself depends on `class-validator`, but the developer has not yet used a `class-validator` flag in their file or codebase. One such case is documented in a [type-graphql issue](https://github.com/MichalLytek/type-graphql/issues/150), for example.

### Proposed Solution
Update the error message with correct grammar and add a sentence explaining that validation flags are expected but not present in the code.

**Comment by NoNameProvided** [comment]
The error message is now updated to the following: 

```
No validation metadata found. No validation will be  performed. There are multiple possible reasons:` 
  - There may be multiple class-validator versions installed. You will need to flatten your dependencies to fix the issue.
  - This validation runs before any file with validation decorator was parsed by NodeJS.
```

**Comment by NoNameProvided** [comment]
This feature/fix has been released in `0.14.0`.
